airflow/settings.py

```diff
@@ -498,7 +498,7 @@ def configure_orm(disable_connection_pool=False, pool_class=None):
 
     engine = create_engine(SQL_ALCHEMY_CONN, connect_args=connect_args, **engine_args, future=True)
     async_engine = create_async_engine(SQL_ALCHEMY_CONN_ASYNC, future=True)
-    create_async_session = sessionmaker(
+    session_maker_async = sessionmaker(
```

Comment:
Your name is more explicit, and I understand why legacy names can feel confusing. (We call a session_maker Session, but that'a actually a factory for session, so you substitute your Session constructor with it, anyway that's how I see it).

But for consistency, this should be `AsyncSession`. (same way as `Session` and `NonScopedSession` above)

Reply:
sure

Reply:
to clarify, with this

> that's how sqlalchemy docs and many other documents online recommend to do it

you are saying that the SA docs recommend doing it by storing session factory as AsyncSession, like we did with Session?

Reply:
👍